model/parameterSheet.cc

```diff
@@ -45,6 +47,137 @@ namespace
       Pango::show();
     }
   };
+}
+
+void ParameterSheet::populateItemvector() {
+  itemvector.clear();	
+  minsky().canvas.selection.recursiveDo(&GroupItems::items,
```

Comment:
Shouldn't we doing this recursiveDo on minsky().model (or even minsky().canvas.model)?

Reply:
Yes. Once I can draw several parameters as cropped sheets down the parameter tab, I will apply the recursive do to all parameters on the canvas.